src/Urbit.ts

```diff
@@ -404,12 +404,15 @@ export class Urbit {
     // called if a channel was reaped by %eyre before we reconnected
     // so we have to make a new channel.
     this.uid = `${Math.floor(Date.now() / 1000)}-${hexString(6)}`;
+    this.emit('status-update', { status: 'initial' });
     this.emit('seamless-reset', { uid: this.uid });
```

Comment:
```suggestion
    this.emit('seamless-reset', { uid: this.uid });
    this.emit('status-update', { status: 'initial' });
```

Not sure if it matters in practice or not, but we should probably signal the reset before the initialization. The former causes the latter.